src/bundle/Resources/public/css/variables.css

```diff
@@ -0,0 +1,37 @@
+:root {
+    --ez-base-font-family: "Noto Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+    /* text colors */
+    --ez-color-text-base: #333;
+    --ez-color-text-white: #fff; /* should probably not be called text white */
```

Comment:
in addition, the action bar and discovery bar action button background colors and background color when hover are missing.